docs/Anjay_integrations/ESP32_integration/M5Stick.md

```diff
@@ -0,0 +1,117 @@
+
+To connect the board:
+
+0. Log in to Coiote DM and from the left side menu, select **Device Inventory**.
+0. In **Device Inventory**, click **Add device**.
+0. Select the **Connect your LwM2M device directly via the Management server** tile.
+![Add via Mgmt](images/mgmt_tile.png "Add via Mgmt")
+0. In the **Device credentials** step:
+    - In the **Device ID** enter your board endpoint name, e.g. `test_device`.
+      ![Device credentials step](images/add_mgmt_quick.png "Device credentials step")
+    - In the **Security mode** section, select the **PSK** mode:
+    - In the **Key identity** field, type `test_device`.
+    - In the **Key** field, type the shared secret used in the device-server authentication.  
+0. Click the **Add device** button and **Confirm** in the confirmation pop-up.
+0. In the **Connect your device** step, wait for the board to connect.
+
+## Step 5: Flash the board and run device
+
+Use pre-built binaries to flash the board and provide credentials by flashing the NVS partition binary.
+
+0. Install the `esptool.py`:
+```
+pip install esptool
+```
+
+0. Flash the board:
+```
+esptool.py -b 750000 --chip esp32 write_flash 0x0000 m5stickc-plus.bin
+```
+
+0. Flash the NVS partition binary:
+```
+esptool.py -b 750000 --chip esp32 write_flash 0x9000 nvs_config.bin
+```
+
+Once executed, the device will be reset and run with the configuration you provided.
+
+![Registered device](images/registered_device.png "Registered device")
+
+## M5Stick LwM2M objects
+
+After successful connection to Coiote DM, you can explore the available device objects.
+
+| Target         | Objects
+|----------------|---------------------------------------------
+| ESP32 common   | Security (/0)<br>Server (/1)<br>Device (/3)<br>Firmware Update (/5)<br>WLAN connectivity (/12)
+| M5StickC-Plus  | Push button (/3347)<br>Light control (/3311)<br>Temperature sensor (/3303)<br>Accelerometer (/3313)<br>Gyroscope (/3343)
+
+
+## Upgrade device firmware over the air
+
+To perform a FOTA upgrade, you need an established connection between the M5Stick and Coiote DM (see instructions above).
+
+### Build new firmware version
+
+0. Go to your `anjay-esp32-client` project directory and run:
+```
+idf.py set-target esp32
+idf.py build
+```
+Once executed, check if the binary file has been built in the following path `$PROJECT_DIR/build/anjay-esp32-client/build/anjay-esp32-client.bin`.
+
+### Schedule the upgrade in Coiote DM
+
+0. In your Coiote DM account, select your device in **Device inventory** and click the **LwM2M Firmware** tab.
+![LwM2M firmware tab](images/lwm2m_firmware.png "LwM2M firmware tab")
+0. Click **Schedule new firmware upgrade**.
+0. Click **Upload** to select the binary file from your local drive, select **COAP** in the **Image delivery protocol**, and click **Upgrade**.
```

Comment:
Currently, our client also works with COAPS so I think we can mention about it. We can also point out that you may need to set a `Base URI` during using COAPS in order to provide credentials.